Gopkg.toml

```diff
@@ -52,6 +52,10 @@
   branch = "v2"
   name = "gopkg.in/yaml.v2"
 
+[[constraint]]
+  name = "k8s.io/api"
+  version = "kubernetes-1.8.5"
+
```

Comment:
I had to pin this because of version conflicts. So far master was used which apparently became incompatible with the other pins we have.